scripts/irods/test/test_iput_options.py

```diff
@@ -15,6 +16,7 @@
 from ..configuration import IrodsConfig
 from .. import lib
 from . import session
+from . import ustrings
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [redefinition of unused 'ustrings' from line 6](https://app.codacy.com/manual/trel/irods/pullRequest?prid=4893066)